RxSwift/Schedulers/SerialDispatchQueueScheduler.swift

```diff
@@ -38,6 +38,11 @@ public class SerialDispatchQueueScheduler : SchedulerType {
 
     let configuration: DispatchQueueConfiguration
     
+    /// Constructs new `SerialDispatchQueueScheduler` that wraps `queue`.
+    ///
+    /// - Parameters:
+    ///   - serialQueue: Target dispatch queue.
```

Comment:
This has to be the same format as others: 

- parameter serialQueue: description ...
- parameter leeway: text...